fairlearn/reductions/_moments/utility_parity.py

```diff
@@ -245,7 +249,11 @@ class DemographicParity(UtilityParity):
 
     def load_data(self, X, y, **kwargs):
         """Load the specified data into the object."""
-        super().load_data(X, y, event=_ALL, **kwargs)
+        control_features = kwargs.get(_KW_CONTROL_FEATURES)
+        if control_features is None:
+            super().load_data(X, y, event=_ALL, **kwargs)
+        else:
+            super().load_data(X, y, event=control_features, **kwargs)
```

Comment:
This is the key point - is this all I need to do to add control features. Basically, create new events for each?